.gitignore

```diff
@@ -2,3 +2,4 @@
 /Propel/om/
 /Propel/map/
 /vendor/
+composer.lock
```

Comment:
What's the reason to remove composer.lock from git?

Reply:
Because it is needed only for this library, and not for projects that depend on this library.
https://getcomposer.org/doc/02-libraries.md#lock-file
> For your library you may commit the composer.lock file if you want to. This can help your team to always test against the same dependency versions. However, this lock file will not have any effect on other projects that depend on it. It only has an effect on the main project.
> 
> If you do not want to commit the lock file and you are using git, add it to the .gitignore.

But I can revert this if it still needed.

Reply:
Makes sense. Thanks for explanation!